flink-runtime/src/main/java/org/apache/flink/runtime/throughput/BufferDebloater.java

```diff
@@ -50,6 +54,14 @@ public BufferDebloater(
 
         this.lastBufferSize = maxBufferSize;
         bufferSizeEMA = new BufferSizeEMA(maxBufferSize, minBufferSize, numberOfSamples);
+
+        LOG.debug(
```

Comment:
I don't know does it make sense or not since `BufferDebloaterConfiguration` is the same for all gates but I think it will be nice to have the gate index(or a similar sign of gate) in the log for each recalculation of buffer size which is only possible if we pass the such a thing into `BufferDebloater`. So it makes sense to think about it.

Reply:
I'd be fine with passing the index, for the sake of cleaner debug logs. Especially as each gate might have different buffer sizes.